health_ext_test.go

```diff
@@ -75,7 +46,7 @@ func TestHealthCheckStopBeforeStart(t *testing.T) {
 		})
 		defer cancel()
 
-		ft := newFakeTicker()
+		ft := testutils.NewFakeTicker()
```

Comment:
I've created https://github.com/uber/tchannel-go/pull/674 to move this out separately to reduce noise in this diff (and to add some simple tests to it).

Reply:
Thank you! I will merge and rebase